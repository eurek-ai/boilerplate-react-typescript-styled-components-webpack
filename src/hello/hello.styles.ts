import styled from "styled-components";

export const Head1 = styled.h1.attrs({
    className: "is-size-1"
  })``;
  