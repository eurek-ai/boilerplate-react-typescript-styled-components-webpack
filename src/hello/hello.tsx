import * as React from "react";
import { FunctionComponent } from "react";

import * as Style from "./hello.styles";

export const HelloComponent: FunctionComponent = () => (
    <Style.Head1>Hello world - ReactJS With Typescript, Styled-Components and Webpack!!</Style.Head1>
);
