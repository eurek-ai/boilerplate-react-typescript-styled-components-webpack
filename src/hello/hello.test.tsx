import * as React from "react";
import { shallow } from "enzyme";
import { HelloComponent } from "./hello";
describe("HelloComponent", () => {
    it("should render correctly", () => {
        const component = shallow(<HelloComponent />);
        expect(component).toMatchSnapshot();
    });
});
