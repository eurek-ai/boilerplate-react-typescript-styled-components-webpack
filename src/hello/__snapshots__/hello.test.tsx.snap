// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloComponent should render correctly 1`] = `
<styled.h1>
  Hello world - ReactJS With Typescript, Styled-Components and Webpack!!
</styled.h1>
`;
